canonical/ds25
# Simplify boilerplate tsconfigs

## Done

Simplifies tsconfigs in the boilerplate project by using a single tsconfig file.

## QA

- Checkout this PR
- `bun i`
- `cd apps/boilerplate-react-vite`
- `rm -rf dist && bun run build`
- See that the `dist` folder is recreated properly
- `bun run dev`
- Open boilerplate [locally](http://localhost:5173/), verify the app still loads and works as it should.

### PR readiness check

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
